docs/apache-airflow-providers-cncf-kubernetes/operators.rst

```diff
@@ -30,6 +30,8 @@ you to create and run Pods on a Kubernetes cluster.
   using the
   :ref:`GKEStartPodOperator <howto/operator:GKEStartPodOperator>` operator as it
   simplifies the Kubernetes authorization process.
+  Or
+  :ref:`EksPodOperator <howto/operator:EksPodOperator>` operator for `AWS Elastic Kubernetes Engine <https://aws.amazon.com/eks/>`__.
```

Comment:
I think it requires rephrase the whole paragraph as the paragraphs talks about Google Kubernetes Engine then suggests to use `GKEStartPodOperator` suggesting `EksPodOperator` is not right for Google Kubernetes Engine users.

Reply:
thanks